pkg/virtual/syncer/controllers/apireconciler/apireconciler_controller.go

```diff
@@ -48,13 +50,15 @@ const (
 	byWorkspace    = controllerName + "-byWorkspace" // will go away with scoping
 )
 
+type CreateAPIDefinitionFunc func(logicalClusterName logicalcluster.Name, workspaceClusterName string, spec *apiresourcev1alpha1.CommonAPIResourceSpec) (apidefinition.APIDefinition, error)
```

Comment:
workspaceClusterName -> workloadClusterName?

Reply:
the code is full of workspaceClusterName :D Feel free to create an issue.

Reply:
fixed this instsance